frontend/language/src/nb.json

```diff
@@ -584,13 +584,17 @@
   "receipt_platform.reference_number": "Referansenummer",
   "receipt_platform.sender": "Avsender",
   "receipt_platform.sent_content": "Følgende er sendt inn:",
+  "process_editor.edit_mode": "Redigeringsmodus",
   "process_editor.fetch_bpmn_error_title": "En feil oppstod ved innlasting av BPMN",
   "process_editor.fetch_bpmn_error_message": "Kunne ikke laste inn BPMN for denne appen. Prøv igjen senere.",
   "process_editor.loading": "Laster inn BPMN",
-  "process_editor.view_mode": "Visningsmodus",
-  "process_editor.edit_mode": "Redigeringsmodus",
+  "process_editor.not_found_process_heading": "Ingen tilgjengelige process",
+  "process_editor.not_found_diagram_heading": "Ingen tilgjengelige diagramdata",
+  "process_editor.not_found_process_error_message": "Det finnes ingen prosess definert innenfor PBNM, Du kan verifisere om prosessen finnes i BPNM-filen <a href=\"http://studio.localhost/repos/JamalAlAbdullah/aktiv\">her</a>.",
+  "process_editor.not_found_diagram_error_message": "Vi kan dessverre ikke vise BPNM-diagrammet for øyeblikket. En mulig årsak til dette kan være at BPNM-filen din ikke inneholder et diagram som er riktig definert innenfor bpnm:diagram-taggen. Du kan verifisere om diagram-taggen finnes i BPNM-filen <a href=\"http://studio.localhost/repos/JamalAlAbdullah/aktiv\">her</a>. En løsning kan være å rekonstruere diagrammet ved hjelp av BPNM-verktøyet",
```

Comment:
Suggest that we do not link to the repo, since we are not pointing them to the exact file. Let the text be something like this:

_Vi kan dessverre ikke vise BPNM-diagrammet for øyeblikket. En mulig årsak til dette kan være at BPNM-filen din ikke inneholder et diagram som er riktig definert innenfor bpnm:diagram-taggen. Du kan verifisere om diagram-taggen finnes i BPNM-filen. En løsning kan være å rekonstruere diagrammet på nytt ved hjelp av BPNM-verktøyet._